pypeit_files/keck_mosfire_j2_long.pypeit

```diff
@@ -0,0 +1,57 @@
+# Auto-generated PypeIt file using PypeIt version: 1.8.2.dev95+g9ee533d34
+# 2022-04-09
+
+# User-defined execution parameters
+[rdx]
+  spectrograph = keck_mosfire
+[baseprocess]
+  use_pixelflat=False
+  use_illumflat=False
+
+[calibrations]
+  [[flatfield]]
+     tweak_slits = False
+  [[traceframe]]
+     [[[process]]]
+        use_darkimage = True
+  [[slitedges]]
+    fit_min_spec_length = 0.3
+
+[reduce]
+  [[findobj]]
+     find_trim_edge = 10,10
+
+
+# Setup
+setup read
+    Setup E:
+          decker: LONGSLIT-46x1
+        dispname: J2-spectroscopy
+         filter1: J2
+setup end
+
+# Read in the data
+data read
+ path PATH_TO_RAW_DATA
+|          filename |                 frametype |           ra |         dec |         target |        dispname |        decker | binning |            mjd |    airmass |   exptime | filter1 | lampstat01 |  dithpat | dithpos | dithoff | frameno | calib | comb_id | bkg_id |
+# Flat fields, and darks 
+| m220409_0017.fits |                     trace |        230.0 |        45.0 |     DOME FLATS | J2-spectroscopy | LONGSLIT-46x1 |     1,1 | 59678.24454412 | 1.41291034 |   8.72874 |      J2 |         on |    Stare |  object |     0.0 |      17 |   all |      -1 |     -1 |
+| m220409_0018.fits |                     trace |        230.0 |        45.0 |     DOME FLATS | J2-spectroscopy | LONGSLIT-46x1 |     1,1 | 59678.24475662 | 1.41291034 |   8.72874 |      J2 |         on |    Stare |  object |     0.0 |      18 |   all |      -1 |     -1 |
+| m220409_0019.fits |                     trace |        230.0 |        45.0 |     DOME FLATS | J2-spectroscopy | LONGSLIT-46x1 |     1,1 | 59678.24498093 | 1.41291034 |   8.72874 |      J2 |         on |    Stare |  object |     0.0 |      19 |   all |      -1 |     -1 |
+| m220409_0020.fits |                     trace |        230.0 |        45.0 |     DOME FLATS | J2-spectroscopy | LONGSLIT-46x1 |     1,1 | 59678.24518753 | 1.41291034 |   8.72874 |      J2 |         on |    Stare |  object |     0.0 |      20 |   all |      -1 |     -1 |
+| m220409_0021.fits |                     trace |        230.0 |        45.0 |     DOME FLATS | J2-spectroscopy | LONGSLIT-46x1 |     1,1 | 59678.24540003 | 1.41291034 |   8.72874 |      J2 |         on |    Stare |  object |     0.0 |      21 |   all |      -1 |     -1 |
+| m220409_0022.fits |                      dark |        230.0 |        45.0 |     DOME FLATS | J2-spectroscopy | LONGSLIT-46x1 |     1,1 | 59678.24685211 | 1.41291034 |   8.72874 |      J2 |         on |    Stare |  object |     0.0 |      22 |   all |      -1 |     -1 |
```

Comment:
This should be lampoffflat not dark right? Also, does your lampofflat functionality work with the trace frames. It absolutely should and if it does not we need to implement that.